can_tools/scrapers/official/base.py

```diff
@@ -175,7 +175,7 @@ def _reshape_variables(
         if "vintage" not in data.columns:
             data["vintage"] = self._retrieve_vintage()
 
-        return data
+        return data.drop_duplicates()
```

Comment:
Out of curiosity, in this case are there duplicates in the incoming `data`?  Or is it only after reshaping the variables that we end up with duplicates?

Part of me wonders if it's a bit odd for incoming data to have duplicates and maybe we should try to catch it more upstream (i.e. have the CDC Covid Data Tracker scraper remove the duplicates) and optionally have an assert here to _verify_ there are no duplicates (rather than remove them). I don't feel that strongly though, mostly just wondering. 😁 

On the other hand if the duplicates are being "created" through the process of reshaping variables then removing them here makes total sense.  

In any case, I wonder if you could craft a comment explaining why / how the duplicates end up here (or provide an example) so the casual reader like myself doesn't see this line and think, "Hrm, I wonder why there are duplicates!".  But if you don't think it's easy / useful to do, feel free to disregard!

Reply:
In this case the duplicates are in the incoming data--to be honest I didn't track it back too far, so I'll look for an explanation. 

I think you're right that it's probably best to not silently remove the duplicates. I'll change this to an assert + add the removal in the CDC Tracker scraper.

Reply:
Awesome, thanks.  If it ends up being a pain to track down / whatever, you could also take the middle-ground and add a comment like:

> # TODO(smcclure17): Not 100% sure why we need this drop_duplicates() but the CDC Covid Data Tracker scraper is ending up with duplicate rows for some states.

or something along those lines. 😁

Reply:
There seem to be rows that are almost duplicated in the original data, then become duplicates when we drop other columns. It only seems to happen with rows that are all zeroes, so I'm not super concerned. I'll add a note

<img width="1163" alt="image" src="https://user-images.githubusercontent.com/55333380/134093037-121512e5-eb6e-4a2a-9e6e-2ae725508459.png">

<img width="1142" alt="image" src="https://user-images.githubusercontent.com/55333380/134093412-825c96f9-43b5-4395-9e84-42ee331867d8.png">

Reply:
having an `assert` in `_reshape_variables` caused problems because the duplicates were created inside that method, so I instead just added an optional parameter to drop the duplicates if specified